src/components/Pages/Unit/widgets/Recommended/Main/UseRecommendedUnits.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectors } from '../../../../../../redux/modules/recommended';
+import { selectors as mdbSelectors } from '../../../../../../redux/modules/mdb';
+
+// A custom hook to get loaded recommended
+export default () => {
```

Comment:
Please call this useRecommendedUnits and return 
`export default useRecommendedUnits ` in a separate line.

Reply:
Done